LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
OK. I'm not advocating for a significant policy change here, so if AGPL matches our past work then let's go with that